.github/workflows/release.yml

```diff
@@ -126,6 +126,17 @@ jobs:
           node-version-file: '.nvmrc'
           cache: 'npm'
       - uses: ./.github/actions/setup
+      - name: check PR number
+        id: action
+        uses: kamatama41/get-pr-number-action@v0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: save PR number
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
I think you are not using this env variable inside the script

Reply:
which env variable do you mean?
GITHUB_TOKEN is a secret stored in secret settings of the repo
github.token is managed by github itself
PR_NUMBER is used inside the script